tests/integration/targets/aws_s3_bucket_info/defaults/main.yml

```
@@ -1,6 +1,6 @@
 ---
-name_pattern: "testbucket-ansible-integration"
+name_pattern: "tb"
 
 testing_buckets:
-  - "{{ resource_prefix }}-{{ name_pattern }}-1"
-  - "{{ resource_prefix }}-{{ name_pattern }}-2"
+  - "{{ resource_prefix | hash('md5') }}-{{ name_pattern }}-1"
+  - "{{ resource_prefix | hash('md5') }}-{{ name_pattern }}-2"
```

```suggestion
  - "ansible-test-{{ resource_prefix | hash('md5') }}-{{ name_pattern }}-1"
  - "ansible-test-{{ resource_prefix | hash('md5') }}-{{ name_pattern }}-2"
```

While hashes are fine in the CI account, people also run these tests in their own accounts.  We've got the spare characters (37 vs 63), so use ansible-test (to link it to ansible integration tests) and info to link it to the bucket_info tests

---

good idea.